### PR TITLE
SIMD-0093:  Disable bpf loader (V2) instructions 

### PR DESCRIPTION
**Pull request by HaoranYi**
Add disable bpf loader (V2) instructions simd

### CODE REVIEW

**Review by lheeger-jump — APPROVED** [review]
Approved

File: proposals/0093-disable-bpf-loader-instructions.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+category: Standard
+type: Core
+status: Draft
+created: 2023-12-13
+feature: https://github.com/solana-labs/solana/issues/33970
+---
+
+## Summary
+
+Add a new feature to disable Bpf loader V2 program deployment.
```
[the remark this answers is not in the document]
Comment (lheeger-jump) [reply proposals/0093-disable-bpf-loader-instructions.md]: We use the same terminology and feature gating structure on FD

**Comment by lheeger-jump** [comment]
I will merge this SIMD, barring core contributor objections, on 11/01/2024, or as soon as labs signs off.

**Review by Lichtso — APPROVED** [review]

**Comment by HaoranYi** [comment]
I don't have write access to this repo. Can someone with write access to merge the PR?

**Comment by jacobcreech** [comment]
Looks like we've reached consensus. Thanks @HaoranYi!

**Comment by pgarg66** [comment]
Looks good!

**Pull request merged**